pyaerocom/io/mscw_ctm/additional_variables.py

```diff
@@ -382,3 +382,47 @@ def calc_concpolyol(concspores):
     concpolyol = concspores.copy(deep=True) * factor
     concpolyol.attrs["units"] = "ug m-3"
     return concpolyol
+
+
+def calc_ratpm10pm25(concpm10: xr.DataArray, concpm25: xr.DataArray) -> xr.DataArray:
```

Comment:
I would like to see some more validation here, in particular an isinstance check on the inputs.

Should this validate that units are compatible?

Reply:
no such tests in the current  about 20 methods in there. I added a unit sanity check anyhow, but the test data has no units so it's in a try block